src/main/java/frc/team2412/robot/commands/shooter/ShootingWhileMovingV2Command.java

```diff
@@ -0,0 +1,66 @@
+package frc.team2412.robot.commands.shooter;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.ShooterSubsystem;
+import static frc.team2412.robot.subsystem.ShooterSubsystem.ShooterConstants.*;
+import frc.team2412.robot.subsystem.TargetLocalizer;
+import frc.team2412.robot.util.ShooterDataDistancePoint;
+import frc.team2412.robot.util.WPILIBInterpolatingTreeMap;
+
+public class ShootingWhileMovingV2Command extends CommandBase {
+    private final ShooterSubsystem shooter;
+    private final TargetLocalizer localizer;
+
+    private static WPILIBInterpolatingTreeMap<Number, Double> timeOfFlight = new WPILIBInterpolatingTreeMap<Number, Double>();
+
+    static {
+        timeOfFlight.put(80, 1.0);
```

Comment:
Where did these values come from?

Reply:
I made them up 👍, should get replaced by actual ones. Made em up just for testing purpose